pkg/server/server_sql.go

```diff
@@ -837,6 +837,7 @@ func newSQLServer(ctx context.Context, cfg sqlServerArgs) (*SQLServer, error) {
 		RangeStatsFetcher:        rangeStatsFetcher,
 		AdmissionPacerFactory:    cfg.admissionPacerFactory,
 		ExecutorConfig:           execCfg,
+		RootSQLMemoryPoolSize:    cfg.MemoryPoolSize,
```

Comment:
is this a noop for sql? sorry I'm not sure I follow why this is needed..

Reply:
This should be. This allows the root SQL memory pool size to be available during the restore processor planning, since the amount of memory we want is limited to a fraction of that. I don't see anything else that exposes this value during restore, but maybe I missed something.